test/e2e/autoscaling_utils.go

```diff
@@ -204,7 +205,8 @@ func (rc *ResourceConsumer) CleanUp() {
 	expectNoError(rc.framework.Client.Experimental().HorizontalPodAutoscalers(rc.framework.Namespace.Name).Delete(rc.name, api.NewDeleteOptions(0)))
 }
 
-func runServiceAndRCForResourceConsumer(c *client.Client, ns, name string, replicas int) {
+// memLimit argument is in megabytes
+func runServiceAndRCForResourceConsumer(c *client.Client, ns, name string, replicas, cpuLimit, memLimit int) {
```

Comment:
Why cpuLimit, memLimit are not int64? In this case you will avoid conversion below